src/components/common/stat/Stat.js

```diff
@@ -886,15 +887,21 @@ class Stat extends PureComponent {
     event.persist();
     this.setState(() => ({
       inputValue: event.target.value,
-    }));
+    }), this.propagateInputChange);
   };
 
   handleSuffixChange = value => {
     this.setState((state) => ({
       inputSuffix: _.assign({}, state.inputSuffix, {
         value,
       }),
-    }));
+    }), this.propagateInputChange);
+  };
+
+  propagateInputChange = () => {
+    if (typeof this.props.onInputChange === 'function') {
```

Comment:
Not a blocker, but should we consider using `_.isFunction` here as it's mildly more robust?

Reply:
👍